modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -0,0 +1,214 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-running-cluster-checkups.adoc
+
+:_content-type: PROCEDURE
+[id="virt-checking-cluster-dpdk-readiness_{context}"]
+= Checking cluster readiness to run DPDK applications with zero packet loss
+
+Use a predefined checkup to verify if your {product-title} cluster node can run a virtual machine (VM) with a Data Plane Development Kit (DPDK) workload. The checkup runs traffic between a traffic generator and a VM running a test DPDK application, and checks for packet loss.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have configured the compute node to run DPDK applications on VMs with zero packet loss.
```

Comment:
This checkup could be executed on a single node, but IMO its better to use it to run traffic across at least two nodes.
```suggestion
* You have configured the compute nodes to run DPDK applications on VMs with zero packet loss.
```